src/react-hooks/buildHooks.ts

```diff
@@ -132,6 +235,21 @@ export type MutationHook<D extends MutationDefinition<any, any, any, any>> = ()
   (
     arg: QueryArgFrom<D>
   ) => {
+    /**
+     * Unwraps a mutation call to provide the raw response/error.
+     *
+     * @remarks
+     * If you need to access the error or success payload immediately after a mutation, you can chain .unwrap().
+     *
+     * @example
+     * ```ts
+     * // codeblock-meta title="Using .unwrap"
+     * addPost({ id: 1, name: 'Example' })
+     *   .unwrap()
+     *   .then((payload) => console.log('fulfilled', payload))
+     *   .catch((error) => console.error('rejected', error));
+     * ```
+     */
```

Comment:
```suggestion
     * @example
     * ```ts
     * // codeblock-meta title="Using .unwrap"
     * addPost({ id: 1, name: 'Example' })
     *   .unwrap()
     *   .then((payload) => console.log('fulfilled', payload))
     *   .catch((error) => console.error('rejected', error));
     * ```
     *
     * @example
     * ```ts
     * // codeblock-meta title="Using .unwrap with async await"
     * try {
     *   const result = await addPost({ id: 1, name: 'Example' })
     *   const payload = result.unwrap();
     *   console.log('fulfilled', payload)
     * } catch (error) {
     *   console.error('rejected', error);
     * }
     * ```
     */
```

Reply:
The async example doesn't seem quite right, should it be like so?
```ts
const payload = await addPost({ id: 1, name: 'Example' }).unwrap()
```

Reply:
Uhh, probably :sweat_smile:
I'm not 100% awake yet :rofl:

Reply:
Added this in manually with the above adjustment